easybuild/easyconfigs/l/LAMMPS/LAMMPS-03Mar20-CrayGNU-20.08-PLUMED-cuda.eb

```diff
@@ -0,0 +1,46 @@
+# Contributed by TWR (CSCS)
+easyblock = 'MakeCp'
+
+name = 'LAMMPS'
+version = '03Mar20'
+local_plumedversion = '2.5.1'
+versionsuffix = '-PLUMED-cuda'
```

Comment:
```suggestion
versionsuffix = '-PLUMED-%s-cuda' % local_plumedversion
```

Reply:
I would keep the version of PLUMED in the `versionsuffix`, since you load a specific version in the dependencies and not just the default like `cudatoolkit`.